src/components/Table.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect } from 'react';
+import StarWarsContext from '../context/StarWarsContext';
+import TableBody from './TableBody';
+import getPlanets from '../services/api';
+
+const Table = () => {
+  const { loading, setLoading, setData } = useContext(StarWarsContext);
+  async function fetchBookList() {
+    await getPlanets().then((response) => {
+      setData(response.results);
+    });
+    setLoading(false);
+  }
```

Comment:
```suggestion
  function fetchBookList() {
    getPlanets().then((response) => {
      setData(response.results);
    });
    setLoading(false);
  }
```

Como você ja esta usando a callback com .then() para tratar a promise, não precisa do await e async